### PR TITLE
Bump version to 4.9.2

### LINKED ISSUE #5730: Support new Wazuh version 4.9.2 in wazuh-qa repository
## Description

This issue attempts to support a new version 4.9.2 in the wazuh-qa repository.

## Tasks

- [x] Bump 4.9.2 branch to 4.9.2 version on wazuh-qa repository, including revision and changelog. https://github.com/wazuh/wazuh-qa/pull/5731

### PR DESCRIPTION
**Pull request by pro-akim**
|Related issue|
|---|
| #5730 |

## Description
Bump 4.9.2 branch to 4.9.2 version on wazuh-qa repository, including revision and changelog.

### CODE REVIEW

File: version.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-    "version": "4.9.1",
-    "revision": "40910"
+    "version": "4.9.2",
+    "revision": "40911"
```
Comment (Rebits) [inline version.json]: Maintain the revision to 40910 please

File: deps/wazuh_testing/wazuh_testing/qa_docs/schema.yaml [hunk 2]
```diff
@@ -206,6 +206,7 @@ predefined_values:
     - 4.8.2
     - 4.9.0
     - 4.9.1
+    - 4.8.2
```
Comment (Rebits) [inline deps/wazuh_testing/wazuh_testing/qa_docs/schema.yaml]:
```suggestion
    - 4.9.2
```

**Review by Rebits — CHANGES_REQUESTED** [review]

File: version.json (hunk 1 above)
Comment (pro-akim) [reply version.json]: Fixed https://github.com/wazuh/wazuh-qa/pull/5731/commits/da2e68400233225450fc893449cfb084d060e574

File: deps/wazuh_testing/wazuh_testing/qa_docs/schema.yaml (hunk 2 above)
Comment (pro-akim) [reply deps/wazuh_testing/wazuh_testing/qa_docs/schema.yaml]: Fixed https://github.com/wazuh/wazuh-qa/pull/5731/commits/da2e68400233225450fc893449cfb084d060e574

**Review by Rebits — APPROVED** [review]

**Pull request merged**